plugins/modules/vmware_host_graphics.py

```diff
@@ -0,0 +1,190 @@
+
+                if changed:
+                    if self.module.check_mode:
+                        not_world = '' if self.restart_xorg else 'not '
+                        self.results[host.name]['changed'] = False
+                        self.results[host.name]['msg'] = f"New host graphics settings will be changed to: hostDefaultGraphicsType = \
+                                                            '{current_config.hostDefaultGraphicsType}', sharedPassthruAssignmentPolicy = \
+                                                            '{current_config.sharedPassthruAssignmentPolicy}'. \
+                                                            X.Org will {not_world}be restarted."
+                    else:
+                        try:
+                            hgm.UpdateGraphicsConfig(current_config)
+                            if self.restart_xorg:
+                                hsm.RestartService('xorg')
+                            xorg_status = 'was restarted' if self.restart_xorg else 'was not restarted.'
+                            self.results['changed'] = True
+                            self.results[host.name]['changed'] = True
+                            self.results[host.name]['msg'] = f"New host graphics settings changed to: hostDefaultGraphicsType = \
+                                                                '{current_config.hostDefaultGraphicsType}', sharedPassthruAssignmentPolicy = \
+                                                                '{current_config.sharedPassthruAssignmentPolicy}'. \
+                                                                X.Org {xorg_status}"
+                        except vim.fault.HostConfigFault as config_fault:
+                            self.module.fail_json(
+                                msg=f"Failed to configure host graphics settings for host {host.name} due to : {to_native(config_fault.msg)}"
+                            )
+                else:
+                    self.results[host.name]['changed'] = False
+                    self.results[host.name]['msg'] = 'All Host Graphics Settings have already been configured'
+            else:
+                self.results[host.name]['changed'] = False
+                self.results[host.name]['msg'] = f"Host {host.name} is disconnected and cannot be changed"
+
+        self.module.exit_json(**self.results)
+
+
+def main():
+    """ Main module method"""
+    argument_spec = vmware_argument_spec()
+    argument_spec.update(
+        cluster_name=dict(type='str', required=False),
+        esxi_hostname=dict(type='list', required=False, elements='str'),
+        graphic_type=dict(type='str', default='shared', choices=['shared', 'sharedDirect'], required=False),
+        assigment_policy=dict(type='str', default='performance', choices=['consolidation', 'performance'], required=False),
+        restart_xorg=dict(type='bool', required=False, default=False),
```

Comment:
```suggestion
        cluster_name=dict(type='str'),
        esxi_hostname=dict(type='list', elements='str'),
        graphic_type=dict(type='str', default='shared', choices=['shared', 'sharedDirect']),
        assigment_policy=dict(type='str', default='performance', choices=['consolidation', 'performance']),
        restart_xorg=dict(type='bool', default=True),
```

`required` defaults to `False`, anyway. Additionally, [this should not be used in combination with default](https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#argument-spec).